Editor/EditorSections/SelectTagsSection.cs

```diff
@@ -114,9 +114,12 @@ private bool IsUserAtTheBottomOfScrollView(int numberOfTags)
             return false;
         }
 
-        private IEnumerable<Tag> FilterTagsByNameAndIfNotPresent()
+        private IEnumerable<Tag> FilterUnselectedTagsByNameAndId()
         {
-            return allTags.Where(tag => !IsTagAlreadyPresent(tag) && TagContainsText(tag.name, searchTagText));
+            return allTags.Where(tag =>
+                !IsTagAlreadyPresent(tag) &&
+                (TagContainsText(tag.name, searchTagText) || TagHasSameId(tag.id, searchTagText))
+            );
```

Comment:
let's say I want to search for a tag that is in the range 250 to 259, but I don't remember the id exactly. I'd like to be able to type "25" and have them all visible, instead of going one by one.
```suggestion
            return allTags.Where(tag => !IsTagAlreadyPresent(tag) && TagContainsText($"{tag.id} {tag.name}", searchTagText)
```